pages/dkp/kommander/2.1/projects/project-deployments/continuous-delivery/index.md

```diff
@@ -11,6 +11,12 @@ excerpt: Continuous Deployment
 
 After installing Kommander and [configuring your project and its clusters](../../), navigate to the **Continuous Deployment (CD)** tab under your Project. Here you create a GitOps source which is a source code management (SCM) repository hosting the application definition. D2iQ recommends that you create a secret first then create a GitOps source accessed by the secret.
 
+## Create a Git repository
```

Comment:
this is ambiguous. there are multiple meanings for this - `github repository`, `any general git repository`, `gitops repository` and then there are CRDs named `GitRepository`, `GitopsRepository` etc., It would be nice to clarify this if possible.

Reply:
good point, this should really be any general git repository but not sure how to specify that 🤔. I'll think on it

Reply:
The next paragraph, and the next heading after this one, seem to imply that the customer is creating a GitOps repo....

Reply:
You could create a heading that says something on the order of “Create a general Git repository for application files," if that's accurate. Then your first para sentence 1 should probably read: "To create a GitOps source, you first need to create a general Git repository to contain your application definitions and Flux resources."

Reply:
I like that approach, more precise. what do you think about e0711f60db6b4ad29016e74b9d26d334459de90a?